tensorflow/contrib/makefile/Makefile

```diff
@@ -66,7 +66,8 @@ HOST_LIBS := \
 -lstdc++ \
 -lprotobuf \
 -lpthread \
--lm
+-lm \
+-lz
```

Comment:
Got it.  The rest of this change seems fine and unavoidable until we can get better factor out the zlib input buffer.